pkg/client/unversioned/helper.go

```diff
@@ -208,15 +208,12 @@ func ServerAPIVersions(c *Config) (groupVersions []string, err error) {
 	}
 	client := http.Client{Transport: transport}
 
-	configCopy := *c
```

Comment:
This was intended to ensure that we didn't change original, so the `configCopy` should have been first and then used throughout the method.  Rather than deleting it, can you update to match intent.